### DISCUSSION TITLE
카카오톡 3.2.7.2782 업데이트 이후 문제

### DISCUSSION THREAD

**Discussion by jjungsu0822** [storycraft/node-kakao#387] [Q&A]
노드카카오 v3을 사용중인 이용자입니다. 
얼마전 업데이트 후의 오류를 #351 의 방법대로 해결하였으나 현재 일부에게는 반응하지만 일부에게는 반응하지 않는 문제가 생겼습니다.
오류 로그가 뜨지도 않고 잘 작동하는데 일부 사람들에게만 봇이 반응을 하지 않습니다. (소스문제 x, 단순반응 소스로 테스트했음에도 동일했습니다.)

**Comment by storycraft** [comment]
v3 버전에서 새로 들어온 사용자가 제대로 인식되지 않는 것은 이미 알려진 버그 입니다.
다만 현재 v3은 관리되고 있지 않습니다. v4를 사용해 주세요.

**Reply by jjungsu0822** [reply]
노드 카카오 초보자라서 그런데 혹시 v4 업데이트법이 무엇인지 여쭤봐도 될까요?

**Reply by raravel** [reply]
v3랑 v4는 로직이 다르기 때문에 하나 하나 기능을 맞추는 방법밖에 없습니다. 대략적인 것은 [migration-v3.md](https://github.com/storycraft/node-kakao/blob/stable/migration-v3.md) 를 참고할 수 있습니다.

**Reply by storycraft** [reply]
npm에서 버전을 특별히 정하지않고 설치 할 경우  v4로 설치됩니다
